firmware/data/config.json

```
@@ -1,6 +1,6 @@
 {
     "device": "TStick",
-    "id": 195,
+    "id": 308,
     "author": "Edu Meneses",
     "institution": "SAT/IDMIL (2022)",
     "APpasswd": "mappings",
```

On the comment at the `.gitignore` file, might be good to transform this into a template config file:

```suggestion
    "APpasswd": "tstick_password",
```